lib/salus/scanners/gosec.rb

```diff
@@ -7,26 +7,81 @@
 module Salus::Scanners
   class Gosec < Base
     def run
+      # 'run_from_dirs' specifies a list of subdirs to run salus from
+      # if not specified, then run_from_dir will mimic the original gosec 'run' behavior
+      return run_from_dir if @config['run_from_dirs'].nil?
+
+      @gosec_failed = false
+      @gosec_stderr = ''   # combined stderr on all runs (one for each configured subdir)
+      @gosec_stdout = ''   # ...      stdout ...
+      @gosec_json = {}     # combined json result on all runs
+      run_from_dirs = @config['run_from_dirs'].sort
+      @config.delete('run_from_dirs')
```

Comment:
Any need to restore this after the run_rom_dir blocks completes?  i.e. making this call impotent.

Reply:
Updated to restore the hash.